sample/signalr-server-selfhost/Connections/HubConnectionAPI.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNet.SignalR;
+
+namespace SelfHost
+{
+    public class HubConnectionAPI : Hub
```

Comment:
Are all these functions being used in tests? If not please leave only what's being used. If you plan to use some of them later stash the changes and add them as you need them.